Cargo.toml

```diff
@@ -9,23 +9,24 @@ readme = "README.md"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 [lib]
-crate-type = ["cdylib"]
+crate-type = ["cdylib", "rlib"]
 
 [features]
-default = ["abi3", "mimallocator"]
+default = ["abi3", "mimallocator", "extension-module"]
 abi3 = ["pyo3/abi3-py36"]
 mimallocator = ["mimalloc"]
+extension-module = ["pyo3/extension-module"]
 
 [profile.release]
 lto = "fat"
 codegen-units = 1
 opt-level = 3
 
 [dependencies]
-pyo3 = { version = "0.13.2", features = ["extension-module"] }
+pyo3 = { version = "0.13.2" }
```

Comment:
The default `auto-initalize` feature which is causing cross compilation and PyPy failures should be safe to disable, see https://pyo3.rs/v0.13.2/features.html#auto-initalize

Reply:
Thanks for the tip, that would have taken me a while! I was following the example work-around [here](https://github.com/PyO3/pyo3/issues/340#issuecomment-461514532) and was just going to ignore running tests for these, but will give this a go first, think that'd be a better solution. :+1:

Reply:
To be honest, I didn't even look at the error; assumed it was something else with tests, but see now it won't build b/c of the auto-initialize feature to which I think had done as you mentioned, but maybe I missed? Maybe it's the `rlib`? I'll try again tomorrow. :+1:

Reply:
I’ll take a another look later.

Reply:
Fixed in https://github.com/milesgranger/pyrus-cramjam/pull/46